prog/test/connected_subnets.rb

```diff
@@ -212,5 +212,7 @@ def test_connection(to_connect_ip, connecting, should_fail: false, ipv4: true)
   def start_listening(ipv4: true)
     vm_to_be_connected.sshable.cmd("sudo systemctl stop listening_ipv#{ipv4 ? "6" : "4"}.service")
     vm_to_be_connected.sshable.cmd("sudo systemctl start listening_ipv#{ipv4 ? "4" : "6"}.service")
+
+    nap 1 if vm_to_be_connected.sshable.cmd("sudo systemctl status listening_ipv#{ipv4 ? "4" : "6"}.service").include?("inactive")
```

Comment:
nit: we might use `is-active` similar to https://github.com/ubicloud/ubicloud/blob/009675777a60b492aaacd438487ae6da581c93e7/prog/vm/nexus.rb#L505